app/common/collections/types.py

```diff
@@ -109,6 +109,26 @@ def get_value_for_text_export(self) -> str:
         return self.label
 
 
+class MultipleChoiceFromListAnswer(SubmissionAnswerBaseModel):
+    choices: list[dict[str, str]]
```

Comment:
Can we make this a TypedDict for better intellisense/checking/autocomplete? It should only be dicts with the keys `key` and `label` right?